schwifty/bank_registry/manual_gb.json

```diff
@@ -1,330 +1,386 @@
   },
   {
     "country_code": "GB",
-    "bic": "REVOGB21",
-    "bank_code": "REVO",
-    "name": "REVOLUT LTD",
-    "short_name": "REVOLUT LTD",
-    "primary": true
+    "primary": true,
+    "bic": "PAEDGB21XXX",
+    "bank_code": "PAED",
+    "name": "PAYSEND PLC",
+    "short_name": "PAYSEND PLC"
   },
   {
     "country_code": "GB",
-    "bic": "DEUTDEFFXXX",
-    "bank_code": "DEUT",
-    "name": "DEUTSCHE BANK AKTIENGESELLSCHAFT",
-    "short_name": "DEUTSCHE BANK AKTIENGESELLSCHAFT",
-    "primary": true
+    "primary": true,
+    "bic": "PAYRGB2L",
+    "bank_code": "PAYR",
+    "name": "PAYRNET LIMITED",
+    "short_name": "PAYRNET LIMITED"
   },
   {
     "country_code": "GB",
-    "bic": "MYMBGB2LXXX",
-    "bank_code": "MYMB",
-    "name": "METRO BANK PLC",
-    "short_name": "METRO BANK PLC",
-    "primary": true
+    "primary": true,
+    "bic": "PAYRGB21",
+    "bank_code": "PAYR",
+    "name": "PAYRNET LIMITED",
+    "short_name": "PAYRNET LIMITED"
   },
   {
     "country_code": "GB",
-    "bic": "BOFAGB22XXX",
-    "bank_code": "BOFA",
-    "name": "BANK OF AMERICA, N.A. LONDON",
-    "short_name": "BANK OF AMERICA",
-    "primary": true
+    "primary": true,
+    "bic": "PNBPGB2L",
+    "bank_code": "PNBP",
+    "name": "WELLS FARGO BANK, N.A., LONDON BRANCH",
+    "short_name": "WELLS FARGO BANK"
   },
   {
     "country_code": "GB",
-    "bic": "BUKBGB22XXX",
-    "bank_code": "BUKB",
-    "name": "BARCLAYS BANK UK PLC",
-    "short_name": "BARCLAYS BANK UK PLC",
-    "primary": true
+    "primary": true,
+    "bic": "PRTCGB21",
+    "bank_code": "PRTC",
+    "name": "PREPAY TECHNOLOGIES LIMITED",
+    "short_name": "PREPAY TECHNOLOGIES LIMITED"
   },
   {
     "country_code": "GB",
-    "bic": "BARCGB22",
-    "bank_code": "BARC",
-    "name": "BARCLAYS BANK PLC",
-    "short_name": "BARCLAYS BANK PLC",
-    "primary": true
+    "primary": true,
+    "bic": "PRTCGB21XXX",
+    "bank_code": "PRTC",
+    "name": "PREPAY TECHNOLOGIES LIMITED",
+    "short_name": "PREPAY TECHNOLOGIES LIMITED"
   },
   {
     "country_code": "GB",
+    "primary": true,
     "bic": "RAPHGB41XXX",
     "bank_code": "RAPH",
     "name": "R RAPHAELS AND SONS PLC",
-    "short_name": "R RAPHAELS AND SONS PLC",
-    "primary": true
+    "short_name": "R RAPHAELS AND SONS PLC"
   },
   {
     "country_code": "GB",
-    "bic": "HBUKGB4BXXX",
-    "bank_code": "HBUK",
-    "name": "HSBC UK BANK PLC",
-    "short_name": "HSBC UK BANK PLC",
-    "primary": true
+    "primary": true,
+    "bic": "RBOSGB2LXXX",
+    "bank_code": "RBOS",
+    "name": "THE ROYAL BANK OF SCOTLAND PUBLIC LIMITED COMPANY",
+    "short_name": "THE ROYAL BANK OF SCOTLAND PUBLIC LIMITED COMPANY"
   },
   {
     "country_code": "GB",
-    "bic": "EPMTGB2LXXX",
-    "bank_code": "EPMT",
-    "name": "EPAYMENTS SYSTEMS LIMITED",
-    "short_name": "EPAYMENTS SYSTEMS LIMITED",
-    "primary": true
+    "primary": true,
+    "bic": "REVO",
+    "bank_code": "REVOGB21",
```

Comment:
I think `bank_code` and `bic` are mixed up here.

Reply:
XXX looks like a good bic to me:

Reply:
Yes. But the bank code has to be 4 letters